modules/custom/az_publication/src/Entity/AZQuickstartCitationStyle.php

```diff
@@ -5,56 +5,62 @@
+use Drupal\az_publication\AZQuickstartCitationStyleHtmlRouteProvider;
+use Drupal\az_publication\AZQuickstartCitationStyleListBuilder;
+use Drupal\az_publication\Form\AZQuickstartCitationStyleForm;
+use Drupal\az_publication\Form\AZQuickstartCitationStyleDeleteForm;
 use Seboettg\CiteProc\Exception\CiteProcException;
 use Seboettg\CiteProc\StyleSheet;
 
 /**
  * Defines the Quickstart Citation Style entity.
- *
- * @ConfigEntityType(
- *   id = "az_citation_style",
- *   label = @Translation("Citation Style"),
- *   label_collection = @Translation("Citation Styles"),
- *   label_singular = @Translation("Citation Style"),
- *   label_plural = @Translation("Citation Styles"),
- *   label_count = @PluralTranslation(
- *     singular = "@count Citation Style",
- *     plural = "@count Citation Styles"
- *   ),
- *   handlers = {
- *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
- *     "list_builder" = "Drupal\az_publication\AZQuickstartCitationStyleListBuilder",
- *     "form" = {
- *       "add" = "Drupal\az_publication\Form\AZQuickstartCitationStyleForm",
- *       "edit" = "Drupal\az_publication\Form\AZQuickstartCitationStyleForm",
- *       "delete" = "Drupal\az_publication\Form\AZQuickstartCitationStyleDeleteForm"
- *     },
- *     "route_provider" = {
- *       "html" = "Drupal\az_publication\AZQuickstartCitationStyleHtmlRouteProvider",
- *     },
- *   },
- *   config_export = {
- *     "id",
- *     "label",
- *     "style",
- *     "custom"
- *   },
- *   config_prefix = "az_citation_style",
- *   admin_permission = "administer site configuration",
- *   entity_keys = {
- *     "id" = "id",
- *     "label" = "label",
- *     "uuid" = "uuid"
- *   },
- *   links = {
- *     "canonical" = "/admin/config/az-quickstart/settings/az-publication/style/{az_citation_style}",
- *     "add-form" = "/admin/config/az-quickstart/settings/az-publication/style/add",
- *     "edit-form" = "/admin/config/az-quickstart/settings/az-publication/style/{az_citation_style}/edit",
- *     "delete-form" = "/admin/config/az-quickstart/settings/az-publication/style/{az_citation_style}/delete",
- *     "collection" = "/admin/config/az-quickstart/settings/az-publication/styles"
- *   }
- * )
  */
+#[ConfigEntityType(
+  id: 'az_citation_style',
+  label: new TranslatableMarkup('Citation Style'),
+  label_collection: new TranslatableMarkup('Citation Styles'),
+  label_singular: new TranslatableMarkup('Citation Style'),
+  label_plural: new TranslatableMarkup('Citation Styles'),
+  handlers: [
+    'view_builder' => EntityViewBuilder::class,
+    'list_builder' => AZQuickstartCitationStyleListBuilder::class,
+    'form' => [
+      'add' => AZQuickstartCitationStyleForm::class,
+      'edit' => AZQuickstartCitationStyleForm::class,
+      'delete' => AZQuickstartCitationStyleDeleteForm::class,
+    ],
+    'route_provider' => [
+      'html' => AZQuickstartCitationStyleHtmlRouteProvider::class,
+    ],
+  ],
+  config_export: [
```

Comment:
Generally the config_export key should contain types defined in https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_publication/config/schema/az_citation_style.schema.yml